### PR TITLE
Fix markup and whitespace

### PR DESCRIPTION
**Pull request by niemela**
Minor changes.

### CODE REVIEW

File: spec/2023-07-draft/problem_package_format.md [hunk 1]
```diff
@@ -90,8 +89,8 @@ point in the table below will be used.
 New in version `2023-07`: `.py` files now default to Python 3, and using shebangs are no longer supported; Python 2 has to be explicitly indicated by the `.py2` extension.
 
 <div class="not-icpc">
-### Problem types
 
+### Problem types
```
Comment (jsannemo) [inline spec/2023-07-draft/problem_package_format.md:93]:
```suggestion
### Problem Types
```

File: spec/2023-07-draft/problem_package_format.md [hunk 2]
```diff
@@ -177,19 +177,35 @@ the "typical system default" is what is needed.
...
+multiple of `time_resolution` that satisfies the above inequalities. It is an
+error if no such multiple exists. The `time_resolution` key is ignored if the
+problem provides an explicit time limit (and in particular, the time limit is
+not required to be a multiple of the resolution). Since time multipliers are
+more future-proof than absolute time limits, avoid specifying `time_limit`
+whenever practical.
+
+Contest systems should make a best effort to respect the problem time limit,
+and should warn when importing a problem whose time limit is specified with
+precision greater than can be resolved by system timers.
```
Comment (jsannemo) [inline spec/2023-07-draft/problem_package_format.md:198]: No.

File: spec/2023-07-draft/problem_package_format.md [hunk 3]
```diff
@@ -303,37 +319,42 @@ If a specific order is desired a numbered prefix such as `00`, `01`, `02`,
...
-There may be `testdata.yaml` files within this structure, but they may only contain the key `input_validator_flags`.
-</div>
+
+In the `data/` directory, there may be an `invalid_inputs/` directory
+containing input files that must be rejected by at least one input validator.
+These are meant to only test the input validators, and are not used for
+judging. The rejected input files can be organized into a tree-like structure
+similar to the test data. <div class="not-icpc"> There may be `testdata.yaml`
+files within this structure, but they may only contain the key
+`input_validator_flags`. </div>
```
Comment (jsannemo) [inline spec/2023-07-draft/problem_package_format.md:350]: :(

File: spec/2023-07-draft/problem_package_format.md [hunk 4]
```diff
@@ -303,37 +319,42 @@ If a specific order is desired a numbered prefix such as `00`, `01`, `02`,
 `03`, and so on, can be used.
 
 <div class="not-icpc">
-In each test data group, a file `testdata.yaml` may be placed to specify how the result of the test data group should be computed.
-If a test data group has no `testdata.yaml` file, the `testdata.yaml` in the closest ancestor group that has one will be used.
-If there is no `testdata.yaml` file in the root `data` group, one is implicitly added with the default values.
+   In each test data group, a file `testdata.yaml` may be placed to specify how
```
Comment (eldering) [inline spec/2023-07-draft/problem_package_format.md:322]:
```suggestion
In each test data group, a file `testdata.yaml` may be placed to specify how
```

**Pull request merged**